content/docs/platform/concepts/subscribers.mdx

```diff
@@ -97,4 +100,110 @@ await novu.trigger({
 
 When triggering the workflow, you must specify all required `to` fields per the channels in the workflow. For example, if an email step is included in the workflow, then the `to.email` field must be specified. Similarly if an SMS step is included, the `to.phone` field must be specified.
 
-If you are defining workflows with code, you can use the `subscriberId` to hydrate subscriber data directly from your database or other sources during workflow execution. This is useful when you don't want to store all the subscriber data in Novu.
+If you are defining workflows with code using [@novu/framework](/framework/introduction), you can use the `subscriberId` to hydrate subscriber data directly from your database or other sources during workflow execution. This is useful when you don't want to store all the subscriber data in Novu.
```

Comment:
Should we add here some title to this code based section, it seems like the continuation of the previous one